CONTRIBUTING.rst

```diff
@@ -130,10 +130,10 @@ This helps us keep track of who is working on what.
 7. Create a branch for local development:
 
 New features added to ``pyjanitor`` should be done in a new branch you have based off the latest version of the ``dev`` branch.
-The protocol for ``pyjanitor`` branches for new development is that the ``master`` branch mirrors the current version of ``pyjanitor`` on PyPI,
-whereas the ``dev`` branch is for additional features for an eventual new official version of the package which might be deemed slightly less stable.
-Once more confident in the reliability/suitability for introducing a batch of changes into the official version,
-the ``dev`` branch is then merged into ``master`` and the PyPI package is subsequently updated.
+The protocol for ``pyjanitor`` branches for new development
+the ``dev`` branch is for additional features for an eventual new official version of the package
```

Comment:
I think there is a word or a few words missing here.

Maybe "The protocol for ``pyjanitor`` branches for new development is that the ``master`` branch mirrors the current version of ``pyjanitor`` on PyPI, whereas the ``dev`` branch..."

Reply:
You're right - I botched the deletion of the reference to the `master` branch. I think we're going to just let the `master` branch wither away - `dev` is where all the action has been, and we have the ability to tag a commit as a release, so I think we're good to go.